commit a2577f635de569d9ece4dc042c000a1b5a810ac7
Author: kerkovits <kerkovits@map.elte.hu>
Date:   Thu Mar 23 21:16:02 2023 +0100

    Fix compile-time errors with GCC 13 https://github.com/GrandOrgue/grandorgue/discussions/1434
    
    * Added cstdint where it was neccessary for gcc13.
    
    * Document bugfix
    
    * Fixed typo and error in BUILD.md

diff --git a/BUILD.md b/BUILD.md
index 0c449dca..e263bfcd 100644
--- a/BUILD.md
+++ b/BUILD.md
@@ -25,7 +25,7 @@ You can download the source code archive from GitHub
     1. `RtAudio`: Download [the source archive](https://github.com/thestk/rtaudio/archive/refs/heads/master.zip) end extract the contents of the ``rtaudio-master`` subdirectory from the archive to the ``submodules/RtAudio`` subdirectory of GrandOrgue source tree.
     2. `RtMidi`: Download [the source archive](https://github.com/thestk/rtmidi/archive/refs/heads/master.zip) and extract the contents of the ``rtmidi-master`` subdirectory from the archive to the ``submodules/RtMidi`` subdirectory of GrandOrgue source tree.
     3. `PortAudio`: Download [the source archive](https://github.com/PortAudio/portaudio/archive/refs/heads/master.zip) and extract the contents of the ``portaudio-master`` subdirectory from the archive to the ``submodules/PortAudio`` subdirectory of GrandOrgue source tree.
-    4. `ZitaConvolver`: Download the source archive](https://salsa.debian.org/multimedia-team/zita-convolver/-/archive/master/zita-convolver-master.zip) and extract the contents of the ``portaudio-master`` subdirectory from the archive to the ``submodules/ZitaConvolver`` subdirectory of GrandOrgue source tree.
+    4. `ZitaConvolver`: Download [the source archive](https://salsa.debian.org/multimedia-team/zita-convolver/-/archive/master/zita-convolver-master.zip) and extract the contents of the ``zita-convolver-master`` subdirectory from the archive to the ``submodules/ZitaConvolver`` subdirectory of GrandOrgue source tree.
 
 ## Building for Linux on Linux
 1. Make sure that GrandOrgue source tree has been extracted to some subdirectory ``<GO source tree>``
diff --git a/src/core/GOHash.h b/src/core/GOHash.h
index 1d1aeb9d..feae0866 100644
--- a/src/core/GOHash.h
+++ b/src/core/GOHash.h
@@ -8,6 +8,7 @@
 #ifndef GOHASH_H
 #define GOHASH_H
 
+#include <cstdint>
 #include <wx/string.h>
 
 #include "contrib/sha1.h"
diff --git a/src/core/archive/GOArchiveFile.h b/src/core/archive/GOArchiveFile.h
index 0564fcd7..6a58b020 100644
--- a/src/core/archive/GOArchiveFile.h
+++ b/src/core/archive/GOArchiveFile.h
@@ -8,6 +8,7 @@
 #ifndef GOARCHIVEFILE_H
 #define GOARCHIVEFILE_H
 
+#include <cstdint>
 #include <wx/string.h>
 
 #include <vector>
diff --git a/src/core/config/GOConfigFileWriter.h b/src/core/config/GOConfigFileWriter.h
index 8766f6a9..928c6304 100644
--- a/src/core/config/GOConfigFileWriter.h
+++ b/src/core/config/GOConfigFileWriter.h
@@ -8,6 +8,7 @@
 #ifndef GOCONFIGFILEWRITER_H
 #define GOCONFIGFILEWRITER_H
 
+#include <cstdint>
 #include <wx/string.h>
 
 #include <map>
diff --git a/src/core/midi/GOMidiEvent.h b/src/core/midi/GOMidiEvent.h
index 80634b9e..5e34cafa 100644
--- a/src/core/midi/GOMidiEvent.h
+++ b/src/core/midi/GOMidiEvent.h
@@ -8,6 +8,8 @@
 #ifndef GOMIDIEVENT_H
 #define GOMIDIEVENT_H
 
+#include <cstdint>
+
 #include <vector>
 
 #include "GOTime.h"
diff --git a/src/grandorgue/model/GOTremulant.h b/src/grandorgue/model/GOTremulant.h
index adbfcaa8..52684029 100644
--- a/src/grandorgue/model/GOTremulant.h
+++ b/src/grandorgue/model/GOTremulant.h
@@ -8,6 +8,7 @@
 #ifndef GOTREMULANT_H
 #define GOTREMULANT_H
 
+#include <cstdint>
 #include <wx/string.h>
 
 #include "GOCacheObject.h"
